File: jss-service/src/main/scala/com/aliyun/emr/jss/service/deploy/worker/PartitionLocationWithDoubleChunks.scala
package com.aliyun.emr.jss.service.deploy.worker

import com.aliyun.emr.jss.protocol.PartitionLocation

class PartitionLocationWithDoubleChunks(
  val location: PartitionLocation,
  val doubleChunk: DoubleChunk
) extends PartitionLocation(location)

File: jss-service/src/main/java/com/aliyun/emr/jss/service/deploy/worker/PartitionLocationWithDoubleChunks.java
package com.aliyun.emr.jss.service.deploy.worker;

import com.aliyun.emr.jss.protocol.PartitionLocation;

public class PartitionLocationWithDoubleChunks extends PartitionLocation {
    private DoubleChunk doubleChunk;

    public PartitionLocationWithDoubleChunks(
        PartitionLocation partitionLocation,
        DoubleChunk doubleChunk
    ) {
        super(partitionLocation);
        this.doubleChunk = doubleChunk;
    }

    public DoubleChunk getDoubleChunk() {
        return doubleChunk;
    }
}
